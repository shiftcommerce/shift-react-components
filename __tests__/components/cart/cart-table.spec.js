// Libraries
import React from 'react'

// Components
import CartTable from '../../../src/components/cart/cart-table'

test('renders correct child elements and classnames', () => {
  const label = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit'
  const className = 'dummy-classname'

  // act
  const wrapper = mount(
    <CartTable className={className}>
      { label }
    </CartTable>
  )

  // assert
  expect(wrapper).toMatchSnapshot()
  expect(wrapper).toIncludeText(label)
  expect(wrapper.find('section')).toHaveClassName(className)
})
