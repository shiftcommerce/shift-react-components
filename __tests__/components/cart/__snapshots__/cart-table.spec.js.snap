// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct child elements and classnames 1`] = `
<CartTable
  className="dummy-classname"
>
  <section
    className="dummy-classname c-cart-table"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit
  </section>
</CartTable>
`;
